hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieFileSliceReader.java

```diff
@@ -32,32 +33,31 @@
 import org.apache.avro.Schema;
 
 import java.io.IOException;
-import java.util.Iterator;
 import java.util.Map;
 import java.util.Properties;
 
 public class HoodieFileSliceReader<T> extends LogFileIterator<T> {
```

Comment:
This class is going to be removed and all the usage will be replaced by `HoodieFileGroupReader`.  So fixing the close behavior in a simple way in this PR.